snap/snapcraft.yaml

```diff
@@ -0,0 +1,49 @@
+name: charmed-openstack-exporter
+base: core22
+summary: OpenStack Exporter for Prometheus
+description: |
+  The OpenStack exporter, exports Prometheus metrics from a running OpenStack
+  cloud for consumption by prometheus.
+adopt-info: openstack-exporter
+confinement: strict
+grade: stable
+
+apps:
+  openstack-exporter:
+    command: bin/openstack-exporter
+    plugs: [home, network, network-bind]
+  service:
+    command: bin/service
+    daemon: simple
+    install-mode: disable
+    restart-condition: on-abnormal
+    plugs:
+      - home
+      - network
+      - network-bind
+      - etc-openstack
+
+parts:
+  openstack-exporter:
+    plugin: go
+    source: https://github.com/openstack-exporter/openstack-exporter.git
+    source-type: git
```

Comment:
I think it's better add `source-tags`, but it depends on how do we want to do the release (always latest change or use stable one)

Reply:
In the current case we need to use the latest change because the last stable one doesn't include the cache introduction

Reply:
got it